pyproject.toml

```diff
@@ -54,7 +54,7 @@ addopts = "--strict-markers -m 'not daily_regression and not weekly_regression a
 
 # by default, tests should be fast.
 # for slower tests, use @pytest.mark.timeout with a higher timeout (specified in seconds)
-timeout = 2
+timeout = 5
```

Comment:
Curious why the timeout increase? Are tests failing with a default timeout of 2?

Reply:
I might have done that accidentally -- reverting.